File: tests/fixtures/java/CleanCode.java
package com.example;

public class CleanCode {
    private int count;

    public CleanCode(int count) {
        this.count = count;
    }

    public int add(int a, int b) {
        return a + b;
    }

    public int getCount() {
        return this.count;
    }
}

File: tests/fixtures/csharp/CleanCode.cs
using System;

namespace Example
{
    public class Calculator
    {
        public int Count { get; set; }

        public int Add(int a, int b)
        {
            return a + b;
        }

        public void Display()
        {
            Console.WriteLine("Count: " + Count);
        }
    }
}
